### DISCUSSION TITLE
Technical troubleshooter for contributors

### DISCUSSION THREAD

**Discussion by m0ksem** [epicmaxco/vuestic-ui#1104] [Ideas]
I think we need to create smth that will help us solve some problems during contributions. For example, if we want to use vuestic-ui as `yarn link` we need to turn off symlinks in vue.config.js because there will be an issue with incorrect vue types. I want to save it somewhere so every contributor will be able to find answers to frequent questions.

We can have this page in docs or maybe somewhere else.

**Comment by asvae** [comment]
[Notion page](https://www.notion.so/epicmax/Vuestic-Onboarding-02eb88911c6149368a0fae99ed0eaad0) - would be a perfect place for that.
